src/io/flutter/vmService/ServiceExtensions.java

```diff
@@ -55,22 +61,23 @@
   public static final ToggleableServiceExtensionDescription<Double> slowAnimations =
     new ToggleableServiceExtensionDescription<>(
       "ext.flutter.timeDilation",
+      "Slow Animations",
       5.0,
       1.0,
       "Disable Slow Animations",
       "Enable Slow Animations");
 
-  public static final ToggleableServiceExtensionDescription<String> togglePlatformMode =
-    new ToggleableServiceExtensionDescription<>(
+  public static final ServiceExtensionDescription<String> togglePlatformMode =
+    new ServiceExtensionDescription<>(
       "ext.flutter.platformOverride",
-      "iOS",
-      "android",
-      "Toggle Platform",
-      "Toggle Platform");
+      "Override Target Platform",
+      new ArrayList<>(Arrays.asList("iOS", "android", "fuchsia")),
```

Comment:
change the type of the API to List from ArrayList and you can avoid the ArrayList constructor calls.